perl6.html.markdown

```diff
@@ -75,7 +75,7 @@ say @array; #=> a 6 b
 #  except they get "flattened" (hash context), removing duplicated keys.
 my %hash = 1 => 2,
            3 => 4;
-my %hash = autoquoted => "key", # keys get auto-quoted
+my %hash = foo => "bar", # keys get auto-quoted if they're simple enough
```

Comment:
Updated to remove "if they're simple enough." Originally, I wanted to clarify that not just any key would get autoquoted, but explaining that becomes too verbose. I think in the context of a short tutorial that can indeed be omited.